ReleaseNotes.md

```diff
@@ -1,17 +1,30 @@
-### New in 0.17.0 (released TBD)
+### New in 0.17.0 (released 2015/12/07)
 
+* New: `NewRepositoryWebHook` helper class useful for creating web hooks - via #917 @alfhenrik
```

Comment:
The convention with previous release entries has been "#PR via @author" - could you bring the #PR forward for each item?